src/SlotInterface.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Drupal\acquia_lift;
```

Comment:
Nah, not feelings at all this time haha. It's a very common convention that [an entity](https://github.com/drupal/drupal/blob/4d4be42a32d8541df122b7658330a500525c7b51/core/lib/Drupal/Core/Entity/Entity.php) and its [entity interface](https://github.com/drupal/drupal/blob/4d4be42a32d8541df122b7658330a500525c7b51/core/lib/Drupal/Core/Entity/EntityInterface.php) share the same physical directory and namespace.